react/react1/week2/react1-week2/src/Main.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useState } from "react";
+import TodoHeader from "./components/TodoHeader";
+import TodoList from "./components/TodoList";
+import './TodoListStyle.css'
+
+const todoArray = [
+    {
+      id: 1,
+      description: "Get out of bed",
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+    },
+  ];
+
+
+
+
+function Main(){
+    const [todos, setTodos]=useState(todoArray)
+    const buttonClick =()=>{
+        const todoArrayLen =todos.length
+        const addTodoList=[
+            {
+              id: todoArrayLen +1,
+              description: `Random text ${todoArrayLen +1}`,
+            }          
+                  ];
+        const nextTodoList=todos.concat(addTodoList)
```

Comment:
a more elegant way of adding a new item to the array is destructuring it and simply typing the new item, fx:

```
setState(...state, {
id: Date.now(),
todo: "review Hanna's PR"
})
```

Reply:
I dont understand this part, how can I add more todo items here when I tried this one 
const addTodoItem=[
            {
              id: Date.now(),
              description: `review Haile's PR`,
            }

Reply:
and whenever I click the add bottom it gives me the same des how can I make different multiple todos items ?

Reply:
Your objective this week is to create multiple todo/tasks, not different ones. 
They need to have different IDs but their content can be the same. 
This is what we call hardcoded todo.
You'll probably learn in the future how to update the todo content.